tests/data_ingestion/test_fhir_ingestion.py

```
@@ -0,0 +1,304 @@
+            "active": True
+        }
+        events = parse_patient_resource_to_event(patient_data_no_meta)
+        # Current behavior: if no timestamp, no event.
+        self.assertEqual(len(events), 0, "Event should not be created if meta.lastUpdated is missing and no fallback.")
+
+    def test_parse_patient_resource_invalid_structure(self):
+        """
+        Test parsing an invalid Patient resource structure (e.g., missing required fields for Patient model).
+        This should be caught by Patient.model_validate()
+        """
+        invalid_patient_data = {
```

Ruff (0.11.9)

Local variable `invalid_patient_data` is assigned to but never used

Remove assignment to unused variable `invalid_patient_data`

(F841)